wordpress-mobile/gutenberg-mobile
# Image Size improvements Mark I

This PR updates the gutenberg ref to test https://github.com/WordPress/gutenberg/pull/18627

Fixes 4 and 5 from https://github.com/wordpress-mobile/gutenberg-mobile/issues/1593

More info in the gutenberg side PR: https://github.com/WordPress/gutenberg/pull/18627

**To test:**

- On WPiOS or WPAndroid.
- Set a new Image block.
- Set add an image to the block (any source should work).
- Switch to HTML mode.
  - IF WPCom: Check that the image URL has the `?w=xxx` property.
  - IF Self-Hosted: Check that the image URL is the proper resized variant: `...img_2782-768x1024.jpg` or similar.

Update release notes:
